indico_custom_profile_fields/client/customFieldForms.jsx

```diff
@@ -10,16 +10,46 @@ import {
 } from 'indico/react/forms';
 import {Translate} from 'indico/react/i18n';
 import customFields from './custom_fields.json';
+import indicoAxios from 'indico/utils/axios';
 
 /*
   Renders custom fields based on the configuration in custom_fields.json
   Supports text, textarea, single_choice (dropdown), and multi_choice (checkboxes)
   fields.
 */
+
+// Helper for flags
+const isoToFlag = country =>
+  String.fromCodePoint(...country.split('').map(c => c.charCodeAt() + 0x1f1a5));
```

Comment:
This seems clever: is it standard ? Also, this only works for ISO 3166-1 alpha-2 (two letters) ?